src/containers/sidebar/index.js

```diff
@@ -1,13 +1,87 @@
 import React from 'react';
-import { Route } from 'react-router-dom'
-import Login from '../login'
-import Register from '../register'
-import Body from '../body'
+import { Route } from 'react-router-dom';
+import Login from '../login';
+import Register from '../register';
+import Body from '../body';
+import Avatar from 'material-ui/Avatar';
+import List from 'material-ui/List';
+import TextField from 'material-ui/TextField';
+import ListItem from 'material-ui/List/ListItem';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import SearchBar from 'material-ui-search-bar';
+import Chip from 'material-ui/Chip';
+import Avi from './avi.jpg';

```

Comment:
Don't you have a folder for all your images?

Reply:
currently we had agreed on keeping the pictures in the same directories as the feature